cmd/coder/sync_windows.go

```diff
@@ -0,0 +1,25 @@
+// +build windows
+
+package main
+
+import (
+	"github.com/spf13/pflag"
+
+	"go.coder.com/cli"
+	"go.coder.com/flog"
+)
+
+type syncCmd struct {
+}
+
+func (cmd *syncCmd) Spec() cli.CommandSpec {
+	return cli.CommandSpec{
+		Name:  "sync",
+		Usage: "[local directory] [<env name>:<remote directory>]",
+		Desc:  "establish a one way directory sync to a remote environment",
+	}
+}
+
+func (cmd *syncCmd) Run(fl *pflag.FlagSet) {
+	flog.Fatal(`"coder sync" is not available for Windows`)
```

Comment:
How hard is it for us to just support this? We just use rsync on both sides of the transfer AFAIK, so as long as the user installs rsync manually and has OpenSSH Client feature enabled on Windows it shouldn't require that much effort from us to support.

Reply:
Russ already mentioned this actually: https://packages.msys2.org/package/rsync

I think if we provided some quick setup instructions for getting rsync installed and working on Windows it'd be a great bonus.

Reply:
I don't have a strong opinion either way.  @nathanpotter thoughts?